classes/Report/SummaryReport.php

```diff
@@ -149,30 +149,40 @@ public function text(): array
         $res[] = sprintf(' Total: %d', $total);
         $res[] = sprintf(' DKIM or SPF aligned: %s', self::num2percent($rdata['summary']['aligned'], $total));
         $res[] = sprintf(' Not aligned: %s', self::num2percent($rdata['summary']['n_aligned'], $total));
+        $res[] = sprintf(' Quarantined: %s', self::num2percent($rdata['summary']['quarantined'], $total));
+        $res[] = sprintf(' Rejected: %s', self::num2percent($rdata['summary']['rejected'], $total));
         $res[] = sprintf(' Organizations: %d', $rdata['summary']['organizations']);
         $res[] = '';
 
         if (count($rdata['sources']) > 0) {
             $res[] = '## Sources';
             $res[] = sprintf(
-                ' %-25s %13s %13s %13s',
+                ' %-25s %13s %13s %13s %13s %13s',
```

Comment:
25+13+13+13+13+13=90 + 5 spaces
It looks too wide for text mail messages. What do you think about it?

Reply:
I'd rather have smaller columns, but the text headers won't fit. Using 'DKIM OK' instead of 'DKIM aligned' would give some space, but it's less accurate. 'Rejected' could be 'Rej', and 'Quarantined' could be 'Qua'. What do you think would be best?